### DISCUSSION TITLE
What is latest stable release available for ImageMagick

### DISCUSSION THREAD

**Discussion by metaad** [ImageMagick/ImageMagick#6948] [General] [answered]
Hi,

I would like to inquire about the latest stable release available for ImageMagick. All releases mentioned on the release page until version 7.1.1-2 appear to be beta releases in the full changelog.

Could you please clarify if all releases after 7.1.1-2 are beta releases? Additionally, if they are, where can I find the Windows installer for version 7.1.1-2?

Thank you.

**Comment by dlemstra** [comment, marked as answer]
The changelog should be read upside down. It starts with the oldest commits and when you scroll all the way down you will see the latest commit that is tagged with the release tag. The first commit is done to mark the release as a beta until we do a new release.

And you can find the downloads for Windows here: https://imagemagick.org/script/download.php#windows. You can only download the latest version we don't have an archive of older versions.

**Reply by metaad** [reply]
Thank you
